resources/wikia/modules/importScriptHelper.js

```diff
@@ -0,0 +1,60 @@
+define('wikia.importScriptHelper', function() {
+	'use strict';
+
+	var wikiaDomain = '.wikia.com',
+		namespacePrefix = 'MediaWiki',
+		externals = {
+			'db': 'external',
+			'domain': 'url'
+		};
+
+	function getNamespacePrefix() {
+		return namespacePrefix;
+	}
+
+	function isJsPage(scriptName) {
+		return scriptName.substr(scriptName.length - 3) === '.js';
+	}
+
+	function isLocal(scriptName) {
+		return scriptName.indexOf(':') === -1;
+	}
+
+	function isExternal(scriptName) {
+		var externalParts = scriptName.split(':');
+
+		return scriptName.indexOf(':') !== -1 && externalParts.length === 3
+			&& ( isExternalDb(externalParts[0]) || isExternalDomain(externalParts[0], externalParts[1]) );
+	}
+
+	function isExternalDb(prefix) {
+		return prefix === externals.db;
+	}
+
+	function isExternalDomain(prefix, domain) {
+		return prefix === externals.domain && hasWikiaDomain(domain);
+	}
+
+	function hasWikiaDomain(scriptName) {
+		if (scriptName.indexOf('.') !== -1) {
```

Comment:
What about non-English subdomains? i.e. `url:ja.starwars:Foo.js`? Right now, only single subdomains are possible without being the full domain.